test/unit/helpers/util/Expect.js

```diff
@@ -62,7 +62,8 @@ define([
         // back on.
         Q.resetUnhandledRejections();
         window.removeEventListener('error', errListener);
-        done();
+        // Wait a tick to make sure the tests clean up
+        fn.tick().then(done);
```

Comment:
I have a feeling this is a symptom of tests that aren't waiting enough - I had added the "waitForFormError" late in my last PR, and didn't go back to convert all error test ticks.

Can you remove this edit, and In primaryAuth_spec can you convert the error ticks to waitForFormError? I.e. like here:
https://github.com/okta/okta-signin-widget/blob/master/test/unit/spec/PrimaryAuth_spec.js#L1182

I'm hoping that will make these tests no longer flaky. If not, let's talk and we can think about adding another tick. However, we shouldn't add it here - it should go to line 54 (my previous PR did not remove the tick, it moved it before this function). The reason to do the wait before this function is because we were previously ignoring errors (i.e. wait, and then check to see if we have any exceptions, etc).